friends/src/store/reducers/login.js

```diff
@@ -0,0 +1,28 @@
+import * as actions from '../actions/actionType';
+
+const initialState = {
+  login: false,
+  error: '',
+  token: '',
+};
+
+const loginReducer = (state = initialState, action) => {
+  switch (action.type) {
+    case actions.LOGIN_SUCCESS:
+      localStorage.setItem('userToken', action.payload);
```

Comment:
Good job on saving the token on local storage, but you should really try to don't cause side effects inside your reducers.